### ISSUE TITLE
Add --metadata flag to new and upload commands

### ISSUE THREAD

**Issue by ezekg** [keygen-sh/keygen-cli#13]
Will likely be required for https://github.com/keygen-sh/keygen-api/issues/489. Should accept a JSON object string and send it as-is, after parsing it to a JSON object, to the API under the release or artifact `metadata` attribute.

https://github.com/keygen-sh/keygen-cli/blob/4a7120005ffd0ba5d2718d05a2b114aa8777d468/cmd/new.go#L62

The following commands should succeed:

```sh
keygen new --version '1.0.0' --metadata '{"foo":"bar","baz":{"qux":1},"quux":[1,2,3]}'
keygen upload tmp/test.txt --release '1.0.0' --metadata '{"foo":"bar","baz":{"qux":1},"quux":[1,2,3]}'
```

Metadata docs are [here](https://keygen.sh/docs/api/metadata/).

**Comment by syedbarimanjan** [comment]
@ezekg can you provide more info into this.

**Comment by ezekg** [comment]
@syedbarimanjan feel free to ask questions. Should be pretty straight forward.

**Comment by ezekg** [comment]
Closed in #15.
